RileyLinkKit/PumpOps.swift

```diff
@@ -149,6 +149,30 @@ public class PumpOps {
         }
     }
 
+
+    /**
+     Reads clock, reservoir, battery, bolusing, and suspended state from pump
+
+     This operation is performed asynchronously and the completion will be executed on an arbitrary background queue.
+
+     - parameter completion: A closure called after the command is complete. This closure takes a single Result argument:
+     - Success(status): A structure describing the current status of the pump
```

Comment:
nit, but this line and the one below it should be tabbed over so Apple's markdown doc generation shows them as bullet points.